tests/golos.publication_rewards_tests.cpp

```diff
@@ -862,6 +921,27 @@ BOOST_FIXTURE_TEST_CASE(golos_curation_test, reward_calcs_tester) try {
     }
     produce_blocks(golos::seconds_to_blocks(150));
     check();
+
+    BOOST_TEST_MESSAGE("--- checking payout if account is closed");
+    BOOST_TEST_MESSAGE("--- create_message: " << name{_close_acc[0]}.to_string());
+    BOOST_CHECK_EQUAL(success(), create_message({_close_acc[0], "permlink"}));
+    check();
+
+    for (size_t i = 0; i < 10; i++) {
+        BOOST_TEST_MESSAGE("--- " << name{_users[i]}.to_string() << " voted");
+        BOOST_CHECK_EQUAL(success(), addvote(_users[i], {_close_acc[0], "permlink"}, 10000));
+        check();
+    }
+
+    BOOST_TEST_MESSAGE("--- " << name{_close_acc[1]}.to_string() << " voted");
+    BOOST_CHECK_EQUAL(success(), addvote(_close_acc[1], {_close_acc[0], "permlink"}, 10000));
+    check();
+
+    BOOST_CHECK_EQUAL(success(), close_token_acc(_close_acc[0], _token_symbol));
+    BOOST_CHECK_EQUAL(success(), close_vest_acc(_close_acc[1], _token_symbol));
```

Comment:
not sure this works, because you need empty balance (withdraw all vesting) before `close` can do it's work.
to do so:
1. buy vesting
2. start withdraw (to simplify things, set vesting params to allow withdraw all in 1 step)
3. wait (withdraw_time - X) where X is less than post cashout window
4. create post, vote for it
5. wait X, now all vesting will be withdrawn
6. close vesting
7. wait post cashout